common/src/main/java/tachyon/security/PlainSaslServer.java

```diff
@@ -100,7 +100,8 @@ public String getMechanismName() {
         throw new SaslException("AuthorizeCallback authorized failure");
       }
     } catch (AuthenticationException ae) {
-      throw new SaslException("AuthenticationProvider authenticate failed:" + ae.getMessage(), ae);
+      throw new SaslException("AuthenticationProvider authenticate failed: " + ae.getMessage(),
```

Comment:
A blank is added to make the message format looks better. 
Yes, it should be in one line. Updated.